crop_image.py

```diff
@@ -10,23 +9,18 @@
 )
 
 
-RIVER_NAME = "MAP"
```

Comment:
Maybe not delete everything in there ?

Reply:
haha it was not supposed to happen... I forgot that  I touched the file for some tests. Going to solve it